modules/api/src/main/java/org/eclipse/xpanse/api/MonitorApi.java

```diff
@@ -34,6 +35,7 @@
 @RestController
 @Validated
 @RequestMapping("/xpanse")
+@Secured({"admin", "user"})
```

Comment:
is it possible to keep roles in some constants page or enum and then use it everywhere?

Reply:
Ok.